### PR TITLE
Fixed a minor compile issue

### PR DESCRIPTION
**Pull request by pauldeng**
Hi @philwareublox, 

Appreciate your work.
I am trying this repo and may this PR help a bit.

Issues:
```
/home/es/xplriot1/ubxlib_cellular_applications_xplr_iot/applications/common/NTPClient.c:26:10: fatal error: ntpclient.h: No such file or directory
   26 | #include "ntpclient.h"
      |          ^~~~~~~~~~~~~
compilation terminated.

/home/es/xplriot1/ubxlib_cellular_applications_xplr_iot/applications/common/NTPClient.c:27:10: fatal error: ..\include\zephyr\sys\byteorder.h: No such file or directory
   27 | #include "..\include\zephyr\sys\byteorder.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

Reproduce steps:
1. Fresh Ubuntu 20.04
2. ./install_linux
3. ./do -e cellular_tracker build
4. issues above occurred

After Fix:
```
./do -e cellular_tracker build
=== cellular_tracker ===
[3/14] Linking C executable zephyr/zephyr_pre0.elf

[6/14] Linking C executable zephyr/zephyr_pre1.elf

[11/14] Linking C executable zephyr/zephyr.elf
Memory region         Used Size  Region Size  %age Used
           FLASH:      270424 B     916992 B     29.49%
            SRAM:      160392 B       504 KB     31.08%
        IDT_LIST:          0 GB         2 KB      0.00%
[14/14] Generating zephyr/merged.hex
sign the payload
sign the payload

# you can find the compiled file at xplriot1/ubxlib_cellular_applications_xplr_iot/_build/cellular_tracker/zephyr/merged.hex
```

Cheers,
Paul

### CODE REVIEW

**Comment by philwareublox** [comment]
Hi,
Thanks for the comment of the filename. 
The byteorder.h file should be found in the zephyr\include\zephyr\sys folder though.... I will check the correct location, but I don't think we need to copy the file in to common. The project is built on zephyr, so hopefully it can be included correctly. I will check...
Regards,
Phil

File: applications/common/byteorder.h
Comment (philwareublox) [inline applications/common/byteorder.h:1]: We shouldn't copy the zephyr header file to the common folder of this application.

File: applications/common/NTPClient.c [hunk 1]
```diff
@@ -23,8 +23,8 @@
 #include <time.h>
 
 #include "common.h"
-#include "ntpclient.h"
-#include "..\include\zephyr\sys\byteorder.h"
```
Comment (philwareublox) [inline applications/common/NTPClient.c:27]: The correct path should be: 
#include "zephyr\sys\byteorder.h"

and in the include folder searching I have this configured;
${workspaceFolder}/../env/ncs/zephyr/**

**Review by philwareublox — CHANGES_REQUESTED** [review]
Can you revert the byteorder.h change and I'll approve the filename case commit.
Your build system should be able to find the byteorder.h file in its #include search of the zephyr include structure.

**Review by philwareublox — APPROVED** [review]
Thanks.

**Comment by pauldeng** [comment]
Can you please enable the Issues for this repo?
You should be able to enable it from repo settings.
I like to chat with you few changes.
image
Cheers,
Paul

**Comment by philwareublox** [comment]
Done.

**Pull request merged**